app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/MiscActions/EventsMergedData/EventsMergedDataContent.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import {
+  Button, Confirm, Message, Table,
+} from 'semantic-ui-react';
+import getEventsMergedData from '../../../../api/competitionResult/getEventsMergedData';
+import Loading from '../../../../../Requests/Loading';
+import Errored from '../../../../../Requests/Errored';
+import deleteResultsData from '../../../../api/competitionResult/deleteResultsData';
+
+export default function EventsMergedDataContent({ ticketDetails }) {
+  const queryClient = useQueryClient();
+  const [showConfirm, setShowConfirm] = useState();
+  const {
+    ticket: {
+      id: ticketId,
+      metadata: { competition_id: competitionId },
+    },
+  } = ticketDetails;
+
+  const {
+    data: eventsMergedData,
+    isPending: isEventsMergedDataPending,
+    isError: isEventsMergedDataError,
+    error: eventsMergedDataError,
+  } = useQuery({
+    queryKey: ['events-merged-data', ticketId],
+    queryFn: () => getEventsMergedData({ ticketId }),
+  });
+
+  const {
+    mutate: deleteResultsDataMutate,
+    isPending: isDeleteDataPending,
+    isError: isDeleteDataError,
+    error: deleteDataError,
+  } = useMutation({
+    mutationFn: deleteResultsData,
+    onSuccess: (_, { model, roundId }) => {
+      queryClient.setQueryData(
+        ['events-merged-data', ticketId],
+        (previousData) => previousData.map((roundData) => {
+          if (model === 'All') {
+            return {
+              ...roundData,
+              count: {
+                results: 0,
+                scrambles: 0,
+              },
+            };
+          }
+          if (roundData.round_id === roundId) {
+            return {
+              ...roundData,
+              count: {
+                ...roundData.count,
+                [model.toLowerCase()]: 0,
+              },
+            };
+          }
+          return roundData;
+        }),
+      );
+    },
+  });
+
+  if (isEventsMergedDataPending || isDeleteDataPending) return <Loading />;
+  if (isEventsMergedDataError) return <Errored error={eventsMergedDataError} />;
+  if (isDeleteDataError) return <Errored error={deleteDataError} />;
+
+  return (
+    <>
+      <Message warning>
+        Note: The table below shows if there are merged data for Result and Scramble. Importing
+        more data will override the existing inbox data, but not the merged results data. You
+        may remove the currently merged results data using the interface below.
+      </Message>
+      <Table celled striped compact="very" definition>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>Event Round</Table.HeaderCell>
+            <Table.HeaderCell colSpan="2">Has Merged Results</Table.HeaderCell>
+            <Table.HeaderCell colSpan="2">Has Merged Scrambles</Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {eventsMergedData.map((roundData) => (
+            <Table.Row key={roundData.round_id}>
```

Comment:
Try `verticalAlign="middle"` for slightly better text flow

Reply:
Thanks, adding it in `Row` didn't work, but adding in `Cell` worked.